src/gluonts/transform/feature.py

```diff
@@ -87,6 +87,8 @@ class MeanValueImputation(MissingValueImputation):
     """
 
     def __call__(self, values: np.ndarray) -> np.ndarray:
+        if len(values) == 1:
+            return DummyValueImputation()(values)
```

Comment:
Would it make sense to pass a user specified dummy_value when constructing the DummyValueImputation object here? Essentially, all these classes would allow specifying the dummy_value, to be used in extreme cases.

But maybe not urgent and can be addressed later.

Reply:
Yeah, so the `DummyValueImputation` accepts a value that uses to impute. With the current implementation it uses the default. We can always add one more argument to all the imputation methods to override the default but maybe this is too much. If needed we can always include it easily.